src/api/api.ts

```diff
@@ -20,6 +20,11 @@ abstract class SessionResource {
 }
 
 export class UserResource extends SessionResource implements Resource<User> {
+    async fetchCurrentUser(): Promise<User> {
+        const r = await this.session.safeFetch(url`users/`);
```

Comment:
isn't it nicer to have a space between url and the string?

Reply:
This is a JS template string. The convention is to write it as fn\`blabla\`, adding a space inbetween would confuse things, IMO.